src/plugins/intel_cpu/tests/functional/custom/single_layer_tests/instances/x64/matmul.cpp

```diff
@@ -550,6 +550,14 @@ const auto testParams2D_Brgconv1x1_smoke = ::testing::Combine(fullyConnectedPara
 
 INSTANTIATE_TEST_SUITE_P(smoke_FC_2D_Brgconv1x1, MatMulLayerCPUTest, testParams2D_Brgconv1x1_smoke, MatMulLayerCPUTest::getTestCaseName);
 
+const auto testParams2D_Brgconv1x1_binary_po =
+    ::testing::Combine(fullyConnectedParams2D_Brgconv1x1_smoke,
+                       ::testing::Values(MatMulNodeType::FullyConnected),
+                       ::testing::Values(fusingReluScaleShift), // eltwise_relu (to prevent Multiply opt out), binary_mul, binary_add
```

Comment:
Why can't we simply include `fusingReluScaleShift` into `fusingParamsSet2D_Brgemm_smoke`?

Reply:
To avoid unnecessary test instances and have meaningful ones.

Reply:
In this particular case it would enlarge code coverage for inner product as well, which is rather a good thing.

Reply:
It should be done in scope of a separate commit then, so called "Increase test coverage for FC node".
This one is intended to verify the faulty behavior that was found.

Reply:
And it will never be done. If we can keep the number of parameterized test instances lower, then this is generally better. Otherwise what is the idea of parametrized tests then? In this particular scenario it will not impact the overall time significantly, also there will be less test code, and the code coverage will be increased.

Reply:
It still makes sense to extend the other Brgemm tests with `fusingReluScaleShift` postop to extend code coverage.

Reply:
I have created a separate PR
https://github.com/openvinotoolkit/openvino/pull/23444